.github/workflows/haskell.yml

```
@@ -73,6 +75,11 @@ jobs:
       if: ${{ matrix.resolver == 'stack' && github.event_name == 'push' && github.ref == 'refs/heads/master' }}
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: latest
+        release_name: Release ${{ steps.build.outputs.sha_short }}
+        draft: false
+        prerelease: false
```

As discussed on Slack, I modified the workflow quite a bit since you opened the PR. I'd still be happy to merge this if you changes these lines since they related to the previous action that was used here.
```suggestion
      with:
        name: Release ${{ steps.build.outputs.sha_short }}
```
See https://github.com/release-drafter/release-drafter#action-inputs